_posts/2018-02-28-Didacta.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: "Die Didacta – eine erfolgreiche Messewoche"
+date: 2018-02-28
+author: Gina
+abstract: "Kurzer Beitrag darüber, was auf der Didacta Bildungsmesse in Hannover passiert ist."
+thumbnail: /images/blog_images/Didacta-Messestand1.jpg
+image1: /images/blog_images/Didacta-Pitch.jpg
+image2: /images/blog_images/Didacta-Podiumsdiskussion.jpg
+lang: de
+---
+Die Didacta - eine erfolgreiche Messewoche
+============
+Letzte Woche waren wir von Dienstag bis Samstag auf der Didacta in Hannover. Sie gilt als größte Bildungsmesse mit rund 73.000 Besuchern und 840 Ausstellern. Auf der Didacta werden Lernkonzepte von der frühkindlichen Erziehung über die schulische Bildung bis hin zu beruflichen Fort- und Weiterbildungsmöglichkeiten vorgestellt. Erstmals gab es auf der Messe ein Start-Up Valley in Halle 13, bei dem sich 50 Start Ups präsentieren konnten – eins davon waren wir! 
+{% include image.html image=page.image1 %}
+
+
+Mit unserem Messestand wollten wir den Besuchern auf interaktive Weise zeigen, was die senseBox so alles kann und wie die senseBox in den Lehrplan von Schulen integriert werden kann. Dabei konnten die Messebesucher selbst ausprobieren, wie die senseBox und die openSenseMap funktionieren. Außerdem haben wir beim German Start Up Pitch teilgenommen.
```

Comment:
Mit unserem Messestand zeigten wir den Besuchern auf interaktive Weise, die vielfältigen Einsatzmöglichkeiten der senseBox und wie eine erfolgreiche integration in den Lehrplan funktionieren kann.